app/models/concerns/build_schedules_relation.rb

```diff
@@ -0,0 +1,15 @@
+module BuildSchedulesRelation
+  extend ActiveSupport::Concern
+
+  included do
+    after_initialize :build_schedules_relation
+
+    # Ensures that the required Schedules are present in the model.
+    #
+    # It makes the need to do this in controller new actions obsolete.
+    # The model takes care of its requirements itsself, as it should.
+    def build_schedules_relation
+      schedules << Schedule.build unless schedules.any?
```

Comment:
Ah sorry I thought `#build` was a standard AR method, in that case ignore my comment :)